### DISCUSSION TITLE
How to initilialize old micro (without creating new micro)?

### DISCUSSION THREAD

**Discussion by TychoLex42** [deta/cloud-docs#349] [Help] [answered]
Yesterday, I created a new micro. Today, after restarting my computer, I wanted to deploy the local changes with the command `deta deploy first_micro`
I got the follwoing error message:

> Error: no deta micro initilialized, see `deta new --help` to create a micro

As I understand it, `deta new --help` only gives info on how to create a new micro, e.g. "second_micro".
The other available commands didn't seem to fit.
How can I initialize my old first_micro from yesterday? I have already logged in with `deta login`
Sorry for this newbie question. Thank you so much for building deta! :-)

**Comment by TychoLex42** [comment, marked as answer]
The problem was that I was trying to use `deta deploy first_micro` from within the micro folder (C:\Users\...\first_micro). Running the command outside the folder. just by spinning up Windows Power Shell did work!

**Reply by wei886-tw** [reply]
Hi TychoLex42,
I got a same problem, trying to open first_micro folder in Vscode still shows no deta micro initilialized, see deta new --help to create a micro. I'm a Mac user do I have to install Power Shell as well?
